netlink.pl

```diff
@@ -220,6 +220,44 @@ sub good {
 mysystem('ssh', $lnx_r_ssh, 'sysctl', 'net.ipv4.ipfrag_high_thresh=1073741824')
     if ($testmode{fragment4});
 
+# install tcpbench service
+mysystem('ssh', $lnx_r_ssh, 'echo [Unit]
```

Comment:
You can use HERE documents for function arguments.

foo("a", <<EOF, "c");
b
EOF